.github/workflows/call-build-windows.yaml

```diff
@@ -45,14 +45,17 @@ jobs:
         config:
           - name: "Windows 32bit"
             arch: x86
-            vcpkg_triplet: x86-windows-static
-            openssl_root_dir_opt: -DOPENSSL_ROOT_DIR=C:\vcpkg\packages\openssl_x86-windows-static
+            win32openssl: https://slproweb.com/download/Win32OpenSSL-1_1_1q.msi
+            win32openssl_dir: "C:\\Program\ Files\\OpenSSL-Win32"
           - name: "Windows 64bit"
             arch: x64
-            vcpkg_triplet: x64-windows-static
-            openssl_root_dir_opt: ""
+            win32openssl: https://slproweb.com/download/Win64OpenSSL-1_1_1q.msi
+            win32openssl_dir: "C:\\Program\ Files\\OpenSSL-Win64"
     permissions:
       contents: read
+    # Default environment variables can be overridden below
+    env:
+        PATH: c:/Program Files/Git/cmd;c:/Windows/system32;C:/Windows/System32/WindowsPowerShell/v1.0;$ENV:WIX/bin;C:/Program Files/CMake/bin
```

Comment:
What's the reason for this change?

Reply:
This is because another openssl dlls are detected on C:\Strawberry\ or something like that. This detected dlls are harmful for linking.

Reply:
Ah ok, can you add a comment then for it? Otherwise it'll likely get deleted next time someone tries to clean it up :+1: